torchao/_models/mixtral-moe/generate.py

```diff
@@ -255,71 +265,72 @@ def main(
 
     if moe_quant:
         torch._dynamo.config.capture_dynamic_output_shape_ops = True
-        config = None
+        config = MoEQuantConfig(
```

Comment:
minor point but the training config is called `MoETrainingConfig` so it would be more clear to call this `MoEInferenceConfig`, to remove any potential ambiguity present in the name `MoEQuantConfig`